cli/wallet/wallet.go

```diff
@@ -103,12 +107,16 @@ func NewCommands() []*cli.Command {
 		txctx.AwaitFlag,
 		inFlag,
 		&flags.AddressFlag{
-			Name:    "address",
-			Aliases: []string{"a"},
-			Usage:   "Address to use",
+			Name:     "address",
+			Aliases:  []string{"a"},
+			Required: true,
```

Comment:
Is it required? It's marked as optional in the command help.

Reply:
it's not. was misled by `TestSignMultisigTx`, I have changed the usage text to optional. sign command can use the default account.